samples/tab-channel-group-config-page-auth/csharp/Controllers/HomeController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System.Diagnostics;
+using TabAuthentication.Models;
+
+namespace TabAuthentication.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly ILogger<HomeController> _logger;
+
+        private readonly IConfiguration Configuration;
+
+        public HomeController(ILogger<HomeController> logger, IConfiguration configuration)
+        {
+            _logger = logger;
+            Configuration = configuration;
+        }
+
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+        [Route("ConfigureTab")]
+        public IActionResult ConfigureTab()
+        {
+            return View();
+        }
+
+        [Route("AuthStart")]
```

Comment:
Could you please separate out Authentication related code to new controller (AuthenticationController) ?